opensrp-reveal/src/main/res/values/strings.xml

```diff
@@ -217,7 +217,7 @@
     <string name="register_outside_boundary_warning">Warning: this location is beyond the boundaries of the selected operational area. If you continue, the structure may be added to an incorrect area</string>
     <string name="register">Register</string>
     <string name="cancel">Cancel</string>
-    <string name="partially_sprayed">Partially Sprayed</string>
+    <string name="partially_sprayed">Sprayed</string>
```

Comment:
I changed the text here to Sprayed, again in case there is an old event that gets played on the Android client so that it shows up as Sprayed. If the user taps the item and saves the form, it will be updated, but I thought it best to say Sprayed from the beginning to accommodate those old events on stage.